opencsp/common/lib/deflectometry/Surface2DParabolic.py

```diff
@@ -327,3 +328,35 @@ def shift_all(self, v_align_optic_step: Vxyz) -> None:
         self.v_optic_cam_optic += v_align_optic_step
         self.v_screen_points_optic += v_align_optic_step
         self.v_optic_screen_optic += v_align_optic_step
+
+    def save_to_hdf(self, file: str, prefix: str = ''):

```

Comment:
This is more of a preference comment than a "best practices" or "requirements" comment.  I like naming my conversion methods starting with either "to_" or "from_", and then they're easy to identify with the IDE.  What do you think?

Reply:
My current convention is "save_to_xxx" or "load_from_xxx." (I believe I have HDF5 and CSV methods with that convention). But I didn't put too much thought into those names; I'm totally fine switching to the standard you mentioned. Shall we move toa to_csv, form_csv, to_hdf, from_hdf, etc. standard.

Reply:
Let's use to_* and from_*. From some quick googling those seem to be the more standard way of naming conversion methods.

Reply:
Or, simply `save(file_path_name_ext: str)` and throw an error for anything that isn't '.h5'. Either way is good.

Reply:
@bbean23, sounds good! I will likely put this in separate PR so this one doesn't get too cluttered.